tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -574,10 +574,21 @@ module.exports = {
      */
     getExpirationDateFromInputField: async function () {
       let expirationDate
-      await this.waitForElementVisible('@expirationDateField').getValue(
-        '@expirationDateField',
+      await this.waitForElementVisible('xpath', '//*[@data-testid="recipient-datepicker"]')
+      await this.getAttribute(
+        'xpath',
+        '//*[@data-testid="recipient-datepicker"]',
+        'value',
         (result) => {
-          expirationDate = result.value
+          const date = new Date(result.value)
+          const dateString =
```

Comment:
Can't you use string formatting on the date to achieve the same result? Probably makes sense to just append the time (` 00:00:00`), but for the date it looks weirdly handcrafted / hacky.